en/Modules/language-management.md

```diff
@@ -0,0 +1,170 @@
+# Language Management Module
+
+This module implements the Language management system of an application;
+
+* Manage **languages** in the system.
+* Translate texts in the UI.
+* Enable/disable **languages**.
+* Select **default language** in the UI.
+
+See [the module description page](https://commercial.abp.io/modules/Volo.Identity.Pro) for an overview of the module features.
+
+## How to Install
+
+Language management module is pre-installed in [the startup templates](../Startup-Templates/Index). So, no need to manually install it.
+
+## Packages
+
+This module follows the [module development best practices guide](https://docs.abp.io/en/abp/latest/Best-Practices/Index) and consists of several NuGet and NPM packages. See the guide if you want to understand the packages and relations between them.
+
+### NuGet Packages
+
+* Volo.Abp.LanguageManagement.Domain.Shared
+* Volo.Abp.LanguageManagement.Domain
+* Volo.Abp.LanguageManagement.Application.Contracts
+* Volo.Abp.LanguageManagement.Application
+* Volo.Abp.LanguageManagement.EntityFrameworkCore
+* Volo.Abp.LanguageManagement.MongoDB
+* Volo.Abp.LanguageManagement.HttpApi
+* Volo.Abp.LanguageManagement.HttpApi.Client
+* Volo.Abp.LanguageManagement.Web
+
+### NPM Packages
+
+* @volo/abp.ng.language-management
+* @volo/abp.ng.language-management.config
+
+## User Interface
+
+### Menu Items
+
+Language management module adds the following items to the "Main" menu, under the "Administration" menu item:
+
+* **Languages**: Language management page.
+* **Language Texts**: Language text management page.
+
+`LanguageManagementMenuNames` class has the constants for the menu item names.
+
+### Pages
+
+#### Languages
+
+Languages page is used to manage languages in the system. 
+
+![language-management-languages-page](../Images/language-management-languages-page.png)
+
+You can create a new language or edit an existing language in this page:
+
+![language-management-edit-language-modal](../Images/language-management-edit-language-modal.png)
+
+* **Enabled** languages can be selected as the system language.
+
+#### Language Texts
+
+Language texts page is used to manage texts in different languages.
+
+![language-management-language-texts-page](../Images/language-management-language-texts-page.png)
+
+You can translate a text for a language or edit the already existing translation in this page.
+
+![language-management-edit-language-text-modal](../Images/language-management-edit-language-text-modal.png)
+
+## Data Seed
+
+This module adds some initial data (see [the data seed system](https://docs.abp.io/en/abp/latest/Data-Seeding)) to the database when you run the `.DbMigrator` application:
+
+* Creates language records configured using `AbpLocalizationOptions`.
+
+If you want to change the seeded language list, see the next section.
+
+## Options
+
+### AbpLocalizationOptions
```

Comment:
Remove this section, because this is not an option of this module, but an option of the [ABP localization system](https://docs.abp.io/en/abp/latest/Localization).

Also, `options.Languages.Add(new LanguageInfo("en", "en", "English"))` makes developers think that the language list is hard-coded, but actually not.